docs/tools/lab/view-xdr/json-to-xdr.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "JSON to XDR"
+sidebar_label: JSON to XDR
+description: "The JSON to XDR converter transforms JSON objects into base-64 encoded XDR format."
+sidebar_position: 0
+---
+
+# JSON to XDR
+
+## Accessing the JSON to XDR Converter
+
+To access the [JSON to XDR](https://lab.stellar.org/xdr/to) page, select "View XDR" from the left-hand navigation menu, then choose the "JSON to XDR" option from the submenu.
+
+![Lab: JSON to XDR page](/assets/lab/lab-xdr-from-json.png)
+
+## Inputting JSON Data
+
+On this page, you can enter a valid base-64 encoded [XDR](../../../learn/fundamentals/data-format/xdr.mdx) JSON object into the input field. To get sample data, navigate to the "[XDR to JSON](./xdr-to-json.mdx)" page, then use the "Copy JSON" button located below the JSON output to copy a formatted object for testing purposes.
```

Comment:
I think "a valid XDR JSON objet" makes more sense instead of "a valid base-64 encoded XDR JSON object"